setting-up-tugboat/index.md

```diff
@@ -0,0 +1,510 @@
+>
+> The private key you provide here must be in PEM format. The public key will be
+> automatically extracted from the private key you upload.
+
+![Upload an SSH Key](_images/upload-an-ssh-key.png)
+
+> #### Warning:: Uploading your SSH key erases the existing key
+>
+> If you upload your own SSH key, this automatically erases the existing SSH
+> key. If you're using this SSH key anywhere, you'll need to update that when
+> you upload the new key.
+
+#### Delete an SSH key
+
+Need to delete or get rid of an SSH key? Go to
+[Repository Settings](#change-repository-settings) -> **Remote SSH Access** for
+the repository whose key you want to delete, and press the **Generate SSH Key**
+button. Generating a new key permanently erases the old key.
+
+### Authenticate with a Docker Registry
+
+If you want to pull images from Docker registries that require authentication,
+you can manage your authorization credentials from within the repo settings.
+
+![Authenticate with a Docker registry](_images/authenticate_with_a_docker_registry.png)
+
+### Delete the Repository
+
+If you want to delete a repo from your Tugboat project, you'll go into the
+Repository Settings for that repo and press the **Delete Repository** button.
+Deleting a repo from Tugboat does not affect any data in the git provider repo
+connected to it, nor does it delete the Tugboat project that contains the repo.
+
+![Delete repository](_images/delete_repo.png)
+
+> #### Note::User Permissions
+>
+> Only Admin users have the **Delete Repository** option. For more on user
+> permissions, see:
+> [User permission levels explained](../administering-tugboat-crew/user-permission-levels-explained/index.md)
+> .
+
+### Change Repository Settings
+
+Any time you need to make a change to Repository Changes, just:
+
+1. Click your username in the upper right-hand corner, and go to Projects;
+2. Click the **My Projects** link;
+3. Select the project where you want to edit repository settings;
+4. Scroll to the linked repository whose settings you want to change, and click
+   the **Settings** link.
+
+From here, you'll see all the Repository Settings you can modify. If you make
+changes to the settings, don't forget to press the **Save Configuration**
+button!
+
+If you want to dive deeper into any of these settings, you can view the full
+list of [Repository Settings](#repository-settings-optional) and drill down to
+the setting you're curious about.
+
+## Create a Tugboat Config File
+
+The final step in setting up your Tugboat is committing a YAML Config file in
+the source's git repository. Think of the Config file as the engine that powers
+your Tugboat; here's where you specify the services you want to run when your
+Tugboat Preview builds.
+
+The Tugboat Config file should live at: `.tugboat/config.yml` in the branch,
+tag, commit or pull request that is being built. The only exception is when
+you're building from a Base Preview; in that case, the config.yml from the Base
+Preview is used.
```

Comment:
> The only exception is when
 you're building from a Base Preview; in that case, the config.yml from the Base
 Preview is used.

This isn't quite right. The build commands are always taken from the code being built. The init/update steps from the base preview will be used for services with matching names. The exception is if you define a new service for a preview that doesn't exist in the base preview, then init/update from the preview being built will be used.